CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add PostLogger support to MeetingProvider
+- [Demo] Add bandwidth metric display
```

Comment:
I think this is not part of 1.2.0 release, right?

Reply:
Sounds good, I'll update this following our release